components/ledger/cmd/container_test.go

```diff
@@ -12,7 +12,7 @@ import (
 
 	"github.com/ThreeDotsLabs/watermill/pubsub/gochannel"
 	"github.com/formancehq/stack/libs/go-libs/otlp/otlptraces"
-	"github.com/numary/ledger/internal/pgtesting"
+	"github.com/formancehq/stack/libs/go-libs/pgtesting"
```

Comment:
Should we do a pass for all ledger and generally all components imports to have every imports pointing to the stack monorepo ? (Not in this PR of course)

Reply:
Yes, i think we should.
Do you have another idea?